src/ZoneServer/Skills/Handlers/Sapper/Claymore.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Threading.Tasks;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.Monsters;
+using static Melia.Zone.Skills.SkillUseFunctions;
+using Melia.Shared.Data.Database;
+
+namespace Melia.Zone.Skills.Handlers.Sapper
+{
+	/// <summary>
+	/// Handler for the Sapper skill Claymore.
+	/// </summary>
+	[SkillHandler(SkillId.Sapper_Claymore)]
+	public class Claymore : IGroundSkillHandler
+	{
+		private Mob trap;
+
+		/// <summary>
+		/// Handles the skill, creates an trap object on the floor on
+		/// the first usage and explodes it on the second usage attempt
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		/// <param name="designatedTarget"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			if (!caster.Position.InRange2D(farPos, skill.Data.MaxRange))
+			{
+				caster.ServerMessage(Localization.Get("Too far away."));
+				return;
+			}
+
+			if (trap != null)
```

Comment:
This is another case that doesn't work because handlers aren't created for every skill use. I should note that I don't mention every instance of every issue I find, so this one for example exists in most of the new handlers.